tests/libcxx/tests.all

```diff
@@ -1,120 +1,104 @@
-/app/CharT.pass.cpp.exe
```

Comment:
Are the changes in tests/libcxx/tests.all intended?

Reply:
Unfortunately running the tests updates ``tests.all``. Xuejun created an issue for this.

Reply:
I restored all the libcxx test files to the ones in main to remove it from this PR. We will have to figure out why they keep getting updating when you run the tests.